src/openapi/streaming/subscription.js

```diff
@@ -263,6 +292,36 @@ function onUnsubscribeError(referenceId, response) {
 }
 
 /**
+ * Called after modify patch is successful
+ * @param referenceId
+ * @param response
+ */
+function onModifyPatchSuccess(referenceId, response) {
+    if (referenceId !== this.referenceId) {
+        log.error(LOG_AREA, 'Received a response for modify patch a subscription that has afterwards been reset - ignoring');

```

Comment:
this should not be an error.. warning?
I guess it could only happen if we have a reset?